cowprotocol/cowswap
# feat(hackathon-surplus): use prod backend when on prod

# Summary

Use prod backend when on prod

# To Test

1. This PR should work against barn
2. Connect wallet
3. Click on the account
* The surplus should be loaded, if any

Once on staging/prod:
- Prod data should be loaded
- Backend endpoint still not available, will be created tomorrow
